libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -527,26 +527,40 @@ void AC_AutoTune::control_attitude()
 
         float target_max_rate;
         switch (axis) {
-        case ROLL:
+        case ROLL: {
             target_max_rate = MAX(AUTOTUNE_TARGET_MIN_RATE_RLLPIT_CDS, step_scaler*AUTOTUNE_TARGET_RATE_RLLPIT_CDS);
+            const float rll_rate_max_cds = degrees(attitude_control->get_ang_vel_roll_max_rads()) * 100;
+            if (is_positive(rll_rate_max_cds)) {
+                target_max_rate = MIN(target_max_rate, rll_rate_max_cds);
+            }
             target_rate = constrain_float(ToDeg(attitude_control->max_rate_step_bf_roll())*100.0f, AUTOTUNE_TARGET_MIN_RATE_RLLPIT_CDS, target_max_rate);
             target_angle = constrain_float(ToDeg(attitude_control->max_angle_step_bf_roll())*100.0f, AUTOTUNE_TARGET_MIN_ANGLE_RLLPIT_CD, AUTOTUNE_TARGET_ANGLE_RLLPIT_CD);
             abort_angle = AUTOTUNE_TARGET_ANGLE_RLLPIT_CD;
             start_rate = ToDeg(ahrs_view->get_gyro().x) * 100.0f;
             start_angle = ahrs_view->roll_sensor;
             rotation_rate_filt.set_cutoff_frequency(attitude_control->get_rate_roll_pid().filt_D_hz()*2.0f);
             break;
-        case PITCH:
+        }
+        case PITCH: {
             target_max_rate = MAX(AUTOTUNE_TARGET_MIN_RATE_RLLPIT_CDS, step_scaler*AUTOTUNE_TARGET_RATE_RLLPIT_CDS);
+            const float ptch_rate_max_cds = degrees(attitude_control->get_ang_vel_pitch_max_rads()) * 100;
+            if (is_positive(ptch_rate_max_cds)) {
```

Comment:
for naming consistency we should use pitch_rate_max_cds

Reply:
done